app_dart/test/datastore/cocoon_config_test.dart

```diff
@@ -32,4 +35,25 @@ void main() {
       expect(installation['godofredoc/cocoon']['installation_id'], equals('123'));
     });
   });
+
+  group('luciTryBuilders', () {
+    FakeConfig config;
+    setUp(() {
+      config = FakeConfig();
+    });
+    test('returns updated builders when there is a config file change in commit', () async {
+      final List<String> files = <String>['dev/try_builders.json'];
+      final List<LuciBuilder> luciBuilders = await config.getLuciTryBuilders(files, 'cocoon', 'test');
+
+      expect(luciBuilders, <LuciBuilder>[const LuciBuilder(name: 'test', repo: 'cocoon', flaky: false)]);
+    });
```

Comment:
It's hard to understand the test expectations without understanding fake_cocoon_config.dart.

Reply:
Thanks for the comment. Added `cocoonTryBuilderValueTot`, and `cocoonTryBuilderValueCommit` to make test self-readable.